recipe_scrapers/pauladeen.py

```diff
@@ -0,0 +1,52 @@
+import re
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class PaulaDeen(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "pauladeen.com"
+
+    def __init__(self, html, url, best_image=None):
+        # Fix invalid JSON-LD by removing the trailing semicolon before the closing script tag
+        html = re.sub(r"};\s*</script>", "}</script>", html)
+        super().__init__(html, url, best_image)
```

Comment:
This works great! But for consistency with the rest of the library, we usually handle these site-specific JSON-LD fixes by creating a small custom SchemaOrg subclass. It keeps the workaround nicely contained to this scraper. (see [here](https://github.com/hhursev/recipe-scrapers/blob/e89e17a41b0ff1c5330e21728cefd819a96efa2c/recipe_scrapers/chefkoch.py#L7) and [here](https://github.com/hhursev/recipe-scrapers/blob/e89e17a41b0ff1c5330e21728cefd819a96efa2c/recipe_scrapers/leckerschmecker.py#L8)) Let me know what you think?

```suggestion
    class _CustomSchemaOrg(SchemaOrg):
        def __init__(self, page_data):
            # Fix invalid JSON-LD by removing the trailing semicolon before the closing script tag
            page_data = re.sub(r"};\s*</script>", "}</script>", page_data)
            super().__init__(page_data)

    _schema_cls = _CustomSchemaOrg

    @classmethod
    def host(cls):
        return "pauladeen.com"
```